dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/CommunityRestRepository.java

```diff
@@ -47,6 +55,39 @@ public CommunityRestRepository() {
         System.out.println("Repository initialized by Spring");
     }
 
+    @Override
+    protected CommunityRest createAndReturn(Context context) throws AuthorizeException {
```

Comment:
This method/endpoint will likely be accessible to anonymous users as-is. I think we need to access restrict this with an `@PreAuthorize` annotation.  Likely should be:
`@PreAuthorize("hasAuthority('ADMIN')")`

For more details on this annotation, see: https://wiki.duraspace.org/display/DSPACE/REST+Authorization

Reply:
Didn't know about that annotation. Thanks for pointing me in that direction @tdonohue. Thanks for your review!